### DISCUSSION TITLE
Переключение треков работет только, если перед этим установить другое расширение

### DISCUSSION THREAD

**Discussion by kusosuha6vt** [Chimildic/YaMuTools#37] [Помощь с ошибкой] [answered]
Я пробовал 2 раза и, кажется, это правда. Если запустить вкладку с музыкой, установить [это](https://chrome.google.com/webstore/detail/%D1%8F%D0%BD%D0%B4%D0%B5%D0%BA%D1%81%D0%BC%D1%83%D0%B7%D1%8B%D0%BA%D0%B0-%D1%83%D0%BF%D1%80%D0%B0%D0%B2%D0%BB%D0%B5%D0%BD%D0%B8%D0%B5-%D0%BF/gcfheefljlblchcfjlknidfimnfillec?hl=en-US) расширение, сразу же его удалить, а затем усттановить yamutools, то переключение музыки работает до закрытия браузера. После этого не работает. Без первого расширения не заработает совсем.
Браузер: Chromium 98.0.4758.102 Manjaro Linux.
В firefox тоже не работает, но там нет первого расширения

**Comment by Chimildic** [comment, marked as answer]
Расширение YaMuTools не предназначено для управления плеером. И фактически никак не затрагивает работу с горячими клавишами.
